aztk_cli/spark/endpoints/cluster/cluster_debug.py

```diff
@@ -0,0 +1,19 @@
+import argparse
+import typing
+import aztk.spark
+from aztk_cli import config
+
+
+def setup_parser(parser: argparse.ArgumentParser):
+    parser.add_argument('--id', dest='cluster_id', required=True,
+                        help='The unique id of your spark cluster')
+
+    parser.add_argument('--output', '-o', required=True,
```

Comment:
couldn't we make that optional to be something like `aztk_debug/[cluster_id]` by default

Reply:
yeah, that's a good idea. by default, it can be `debug-{cluster-id}/` in the working directory.